projects/canopy/src/lib/pagination/pagination.component.scss

```diff
@@ -0,0 +1,47 @@
+@import '../../styles/mixins';
+
+.lg-pagination {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: wrap;
+  column-gap: var(--space-xxxs);
+  row-gap: var(--space-sm);
+
+  ul {
+    list-style-type: none;
+    display: flex;
+    flex-direction: row;
+    margin: 0;
+  }
+
+  /* stylelint-disable */
+  --button-size: 44px;
+  /* stylelint-enable */
+
+  .lg-pagination__button {
+    width: var(--button-size);
+    height: var(--button-size);
+    border-width: 0;
+    border-radius: 50%;
+    font-size: var(--text-fs--8-size);
+    background-color: transparent;
+
+    &:hover:not(:disabled) {
+      background-color: var(--pagination-button-hover-bg-color);
+      color: var(--pagination-button-hover-color);
+    }
+
+    &:focus,
```

Comment:
What I meant with my previous comment is changing these selectors to be:
```
&:focus-visible,
&:hover:focus-visible,
&:hover:focus-visible:hover {
```

Reply:
done